extensions/cornerstone/src/tools/ImageOverlayViewerTool.tsx

```diff
@@ -0,0 +1,258 @@
+import { metaData, StackViewport, Types } from '@cornerstonejs/core';
+import { utilities } from '@cornerstonejs/core';
+import { BaseTool } from '@cornerstonejs/tools';
+import { guid } from '@ohif/core/src/utils';
+
+interface CachedStat {
+  color: number[];
+  overlays: (unknown & {
+    _id: string;
+    pixelDataRaw: ArrayBuffer;
```

Comment:
Consider making this just the raw overlay data, plus the color, eg:
color: Types.Color;
overlayData: URL;
size: vec2;
position: vec2;

That way the tool is added to the image area once it is actually loaded, and is for a specific image - so in fact you could consider adding it to the measurements panel if desired.